portfolio/src/main/webapp/contact.html

```diff
@@ -11,14 +12,14 @@
     </head>
     <header>
         <div class="navBar">
-        <div id="logo">Yaa Owusu</div>
+        <div id="logo"><a href='.'>Yaa Owusu</a></div>
```

Comment:
What are you trying to do with <a href='.'>?  I have not seen it refer to a dot before, so I'm sincerely asking :)

Reply:
This makes it link back to the home page, but I think the better way is to use / or ./ ?

Reply:
Ah, I see there is not a specific way to do this, many ways work.